cnapy/core.py

```diff
@@ -39,8 +41,33 @@ def efm_computation(model: cobra.Model, scen_values: Dict[str, Tuple[float, floa
         ems = None
     else:
         ems = FluxVectorMemmap('efms.bin', reac_id,
-                                containing_temp_dir=work_dir)
+                               containing_temp_dir=work_dir)
         if len(irrev_backwards_idx) > 0:
             ems.fv_mat[:, irrev_backwards_idx] *= -1
```

Comment:
@axelvonkamp I get here a warning: `'ems.fv_mat' does not support item assignment`